docs/reference/data-sources/trino.md

```diff
@@ -0,0 +1,29 @@
+# Trino source (contrib)
```

Comment:
you'll need to add this to the SUMMARY.md page I think for this to properly show up